fastlane/Fastfile

```diff
@@ -31,16 +33,18 @@ lane :verify do |options|
 end
 
 desc "Build demo app"
-lane :verify_demo do
-    scheme = "iOS Example"
-    output_directory = "fastlane/test_output/" + scheme + "/"
+lane :verify_demo do |options|
+    scheme = options[:scheme]
+    destination = options[:destination]
+    sdk = options[:sdk]
```

Comment:
How does this work if you're not passing in `destination` or `sdk` in `options`?

Reply:
They get ignored when running the actual xcodebuild command as they have no value. It's mostly there from legacy times when they were required, but I've kept them in case in the future they're required again. It was a copy-paste from FormValidatorSwift.